### DISCUSSION TITLE
The status does not progress beyond onboarding.

### DISCUSSION THREAD

**Discussion by stobita** [renovatebot/renovate#31724] [Request Help]
### What would you like help with?

None

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

I reopened an onboarding PR that I had once closed and merged it into the main branch, but when I checked the console, the branch's status did not change to activated and remained in onboarding mode.
I'd like to know if there is a way to move forward with this.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
What is the repository name?
